esp-config/src/generate.rs

```diff
@@ -20,10 +20,19 @@ impl Value {
     fn parse(&mut self, s: &str) -> Result<(), ParseError> {
         *self = match self {
             Value::Bool(_) => match s {
-                "false" => Value::Bool(false),
-                _ => Value::Bool(true), // TODO should existance of the key mean true?
+                "false" | "no" | "n" => Value::Bool(false),
+                "true" | "yes" | "y" => Value::Bool(true),
+                _ => return Err(ParseError("Invalid boolean value")),
```

Comment:
Please include the incorrect value in the error message, I hate guessing :D
```suggestion
                other => return Err(ParseError(format!("Invalid boolean value '{other}'")),
```